ios/brave-ios/Sources/Brave/WebFilters/ContentBlocker/ContentBlockerManager.swift

```diff
@@ -206,6 +209,75 @@ import os.log
     Self.signpost.endInterval("cleaupInvalidRuleLists", state)
   }
 
+  /// Test the rules and find the broken rules, their line numbers and associated errors
+  public func testRules(
```

Comment:
Do we need test method public? With testable imports we should be fine having them on internal visiblity

Reply:
maybe the name is not clear but it has nothing to do with XCTests. This is a method that checks to see if the rules are correct and spits out the invalid lines.